sufia-models/lib/sufia/models/engine.rb

```diff
@@ -30,6 +28,14 @@ class Engine < ::Rails::Engine
       config.max_notifications_for_dashboard = 5
       config.activity_to_show_default_seconds_since_now = 24*60*60
 
+      # Noid identifiers
+      config.enable_noids = true
+      config.noid_template = '.reeddeeddk'
+      config.minter_statefile = '/tmp/minter-state'
```

Comment:
I'd recommend someplace more permanent for people wanting noid minting to be fairly guaranteed to not repeat